cypress.json

```diff
@@ -23,5 +23,10 @@
   "retries": {
     "runMode": 1,
     "openMode": 1
-  }
+  },
+   "component": {
+     "componentFolder": "src",
+     "testFiles": "**/*.test.js",
+     "watchForFileChanges": true
+   }
```

Comment:
Space before the `}` to remove.

Reply:
I think it should be fixed separately :) lets get merge this one :)

Reply:
done